Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WinForms/WebView/WebView.cs

```diff
@@ -423,6 +424,14 @@ public Task<string> InvokeScriptAsync(string scriptName, IEnumerable<string> arg
         /// <inheritdoc />
         public void Navigate(Uri source) => _webViewControl?.Navigate(source);
 
+        /// <inheritdoc />
+        public void Navigate(
+            Uri requestUri,
+            HttpMethod httpMethod,
+            string content = null,
+            IEnumerable<KeyValuePair<string, string>> headers = null) =>
+            _webViewControl.Navigate(requestUri, httpMethod, content, headers);
+
         /// <inheritdoc />
```

Comment:
Doesn't NavigateWithHttpRequestMessage need to be exposed to?

Reply:
NavigateWithHttpRequestMessage is internal to the host, taking a WinRT type and not .NET types. The host has a Navigate method like the WinForms and WPF API surface that performs the translation.